recipe.yaml

```diff
@@ -43,9 +43,10 @@ steps:
     #                   applies input feature transformation during model training and inference.
     transformer_method: steps.transform.transformer_fn
   train:
-    using: estimator_spec
-    # Specifies the method that defines the estimator type and parameters to use for model training
-    estimator_method: steps.train.estimator_fn
```

Comment:
If we get rid of this, should we remove the code from `steps/train.py` since that is not needed anymore?

Reply:
I was thinking about this. I think it is important to show users how that code should be written (even if it is not used in this example). 

Specifically, we need users to return something with estimator params args such as `SGDRegressor(**estimator_params)` in order to pass in hardcoded and tuning params

WDYT? Does it make sense to make an "automl" example and a "estimator_spec" example? @sunishsheth2009